src/test/java/org/jabref/logic/formatter/casechanger/TitleCaseFormatterTest.java

```diff
@@ -10,8 +13,43 @@
  */
 public class TitleCaseFormatterTest {
 
-    private TitleCaseFormatter formatter;
+    private final TitleCaseFormatter formatter = new TitleCaseFormatter();
+
+    private static Stream<Arguments> testData() {
+        return Stream.of(
+                Arguments.of("Upper Each First", "upper each first"),
+                Arguments.of("Upper Each First", "upper eACH first"),
+                Arguments.of("An Upper Each First And", "an upper each first and"),
+                Arguments.of("An Upper Each First And", "an upper each first AND"),
+                Arguments.of("An Upper Each of the and First And",
+                             "an upper each of the and first and"),
+                Arguments.of("An Upper Each of the and First And",
+                             "an upper each of the AND first and"),
+                Arguments.of("An Upper Each of: The and First And",
+                             "an upper each of: the and first and"),
+                Arguments.of("An Upper First with and without {CURLY} {brackets}",
+                             "AN UPPER FIRST WITH AND WITHOUT {CURLY} {brackets}"),
+                Arguments.of("An Upper First with {A}nd without {C}urly {b}rackets",
+                             "AN UPPER FIRST WITH {A}ND WITHOUT {C}URLY {b}rackets"),
+                Arguments.of("{b}rackets {b}rac{K}ets Brack{E}ts",
+                             "{b}RaCKeTS {b}RaC{K}eTS bRaCK{E}ts"),
+                Arguments.of("Two Experiences Designing for Effective Security",
+                             "Two experiences designing for effective security"),
+                Arguments.of("Bibliographic Software. A Comparison.",
+                             "bibliographic software. a comparison."),
+                Arguments.of("Bibliographic Software. {A COMPARISON.}",
+                             "bibliographic software. {A COMPARISON.}"),
+                Arguments.of("{BPMN} Conformance in Open Source Engines",
+                             new TitleCaseFormatter().getExampleInput()));
+    }
 
+    @ParameterizedTest
+    @MethodSource("testData")
+    public void test(String expected, String input) {
+        assertEquals(expected, formatter.format(input));
+    }
+
+    /*
```

Comment:
Is there a reason why these tests are commented out?

Reply:
They are redundant as I already added them as parameterized tests but I didn't know if I should delete them yet since they contain the descriptions of the tests.

Reply:
Then just remove them. I think it's clear from the arguments what is tested